### ISSUE TITLE
referenceStripElement doesn't work?

### ISSUE THREAD

**Issue by mmcclimon** [openseadragon/openseadragon#333]
I'm trying to put a referenceStrip into its own element (I don't want it to overlay on the main image), but I can't get it to work. Neither of the following work:

```
  var viewer = OpenSeadragon({
    id: 'viewer',
    prefixUrl: 'images/',
    tileSources: tileSrcs,
    showReferenceStrip: true,
    referenceStripElement: 'navstrip'
  });

  var viewer = OpenSeadragon({
    id: 'viewer',
    prefixUrl: 'images/',
    tileSources: tileSrcs,
    showReferenceStrip: true,
    referenceStripElement: document.getElementById('navstrip'),
  });
```

Both of them wind up throwing `Uncaught TypeError: Cannot read property 'style' of undefined` at openseadragon.js, line 10305.

I'm pretty sure the problem is in the viewer class: [in the viewer class](https://github.com/openseadragon/openseadragon/blob/master/src/viewer.js#L1500), the `referenceStripElement` property is assigned to the property `id`, but [in the ReferenceStrip class](https://github.com/openseadragon/openseadragon/blob/master/src/viewer.js#L1500), there's no way to actually pass in an element (as far as I can tell). Because `options.id` is set in ReferenceStrip, the `if` at line 75 never fires, and `this.element` is always undefined, thanks to the `$.extend` at line 89.

I _think_ (but am not sure), that allowing an `element` property in the options to ReferenceStrip would fix the the problem, but might cause other problems, so I don't want to submit a PR.

**Comment by VipSaran** [comment]
Why is this still not merged? 5 years... o.O

v2.4.0 still lacks support for custom reference strip container...

**Comment by iangilman** [comment]
Yeah, looks like @mmcclimon walked away from the patch (https://github.com/openseadragon/openseadragon/pull/338) before it was done. @VipSaran would you be up for taking it across the finish line? Doesn't look like there is a lot to do...

**Comment by HemboPagi** [comment]
Hello, after update to the lates vesion od OSD i do not get the error mentioed above anymore, but at the same place new error appears: `[Error] TypeError: undefined is not an object (evaluating 'this.element.tabIndex = 0')
	(anonymous function) (openseadragon.js:16452)`

**Comment by iangilman** [comment]
@HemboPagi There was work in progress to fix this on PR https://github.com/openseadragon/openseadragon/pull/1961, but @msalsbery hasn't updated it in a while. Would you be interested in taking it across the finish line?

**Comment by msalsbery** [comment]
I have a branch with a bunch of reference strip fixes. The last thing on my TODO for that, which I was starting to work on, was the fix for proper support for using a specific parent element 🙄

**Comment by iangilman** [comment]
@msalsbery Nice! It'll be most welcome :)

**Comment by HemboPagi** [comment]
@msalsbery would be ready to share your branch?

**Comment by msalsbery** [comment]
> @msalsbery would be ready to share your branch?

I think I pushed the working branch quite a while ago, but I'm also back working on it. Sorry for the extreme lag!

**Comment by iangilman** [comment]
@HemboPagi I believe the branch in question is https://github.com/openseadragon/openseadragon/pull/1961.

**Comment by msalsbery** [comment]
@iangilman gracias! I’ll sync/update it to latest code if I don’t just finish it
